styleguide/_static/issue-templates/feature.md

```diff
@@ -0,0 +1,47 @@
+---
+name: ✨ Feature
+about: Add/Implement Feature
+title: ✨ <Insert Title>
+labels: feature
+assignees: ''
+
+---
+
+# Summary
+<!---
+A short description of the feature (TL;DR)
+--->
+
+# Details
+<!---
+A detailed description about what should be added, background, links, context information
+and usage example(s) of the feature.
+--->
+
+## Background & Context
+<!---
+What is the motivation for the feature?
+Assessment of the feature. Pros & cons, solutions and decisions concerning the feature (reasoning).
```

Comment:
Maybe add sub-sections:
Design Decisions
Architecture

Reply:
:smile: totally get what you mean! Previous version of the templates have been way more detailed. While thinking more about it, I realized that I'm not 100% sure what the ideal substructure would look like. Therefore I did not want put too many "constraints" on the author(s). My thought was, if at some point it is clear what the best structure would be, we can add it.